.travis/build-deploy.sh

```diff
@@ -60,9 +60,12 @@ elif travis_isPullRequest; then
     log_info "This is a pull-request build"
     ./mvnw verify $MVN_BUILD_FLAGS
 	(
-	    set +e
-	    log_info "Running danger"
-	    bundle exec danger --verbose
+            set +e
+            # if ${TRAVIS_BRANCH} is master, the script will fail since there is local master branch
```

Comment:
Did you mean "if ${TRAVIS_BRANCH} is *not* master, the script will fail since there is *no* local master branch"?

It seems, that after a "git clone", which travis does, a local master branch is always available. But I guess, it's better, to make sure regardless of the branch name.